Octokit/Clients/OrganizationMembersClient.cs

```diff
@@ -37,6 +37,22 @@ public enum OrganizationMembersRole
         Member
     }
 
+    public enum OrganizationMembershipsState
```

Comment:
I'm not 100% sold on this name either, maybe it should be singular `Membership` rather than plural `Memberships`...

Reply:
So I just found [`MembershipState`](https://github.com/octokit/octokit.net/blob/master/Octokit/Models/Response/MembershipState.cs#L9) 🤦‍♂